frontend/src/ui/components/DomainSuccessJumbotron/DomainSuccessJumbotron.tsx

```diff
@@ -28,7 +28,10 @@ export const DomainSuccessJumbotron: React.FC<DomainProps> = (
         <WrappedMessage id={domainStatusText} messages={messages} />
       </h2>
       <div className="domain-name">
-        <p>{props.domain}</p>
+        <p>
+          {props.domain}
+          {props.domainIsExternal === false && <span>.opencraft.hosting</span>}
```

Comment:
Nit: isn't ` === false` redundant? `domainIsExternal` is already a `boolean`.

Could we move this suffix to the configs instead of hardcoding it? It reduces the risk of typos.